### DISCUSSION TITLE
`prPriority` config option docs does not list parents = `packageRules`

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#29185] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

None

## Summary

Should the `prPriority` config option docs have a field for the `parents = packageRules`?

### Background info

Rescuing a review comment from getting lost, and to not mess up the review process for that PR. `@rarkins` said:

> Also yes, `prPriority` should always be part of `packageRules`

- https://github.com/renovatebot/renovate/pull/29178#issuecomment-2121942452

### Renovate docs for `prPriority`

The [Renovate docs, `prPriority` config option](https://docs.renovatebot.com/configuration-options/#prpriority) table says:

| Name | Value |
| --- | --- |
| type | integer |
| default | 0 |

See how the `prPriority` does _not_ list `parents = packageRules`.

### Renovate docs for `allowedVersions`

For example `allowedVersions` has a table that says:

| Name | Value |
| --- | --- |
| type | string |
| parents | packageRules |
| default | null |

## What should we do?

Should we add a `parents = packageRules` field to the code for the `prPriority` config option?

Want me to make an issue to track adding `parents = packageRules` to the `prPriority` config options code?

### Logs (if relevant)

Logs not relevant. Looking at the current code will probably help decide what to do. :wink:

**Comment by rarkins** [comment, marked as answer]
- https://github.com/renovatebot/renovate/pull/29201

**Reply by HonkingGoose** [reply]
@rarkins I suspect we have more config options that have a missing `parents` field in the code.

- Do you want to check for more missing `parents` fields yourself?
- Or do you want me to create an issue to track checking for missing `parents` fields?

I don't know enough to check this stuff myself unfortunately. :see_no_evil:
